src/set_theory/zfc/basic.lean

```diff
@@ -754,6 +754,25 @@ theorem image.mk :
   (image f x).to_set = f '' x.to_set :=
 by { ext, simp }
 
+/-- The range of an indexed family of sets. -/
+noncomputable def range {α : Type u} (f : α → Set.{max u v}) : Set.{max u v} :=
```

Comment:
I think this could do with a brief comment in the docstring about the weird universes

Reply:
Well, it allows for a more general index type without the manual use of `ulift`. Seems obvious enough, but I can still add the comment.

Reply:
I agree with Eric that the doc string be updated to explain the universes before we merge this.